aiidalab_widgets_base/__init__.py

```diff
@@ -27,22 +25,31 @@ def is_running_in_jupyter():
         return False
 
 
-# load the default profile if no profile is loaded, and raise a deprecation warning
-# this is a temporary solution to avoid breaking existing notebooks
-# this will be removed in the next major release
-if is_running_in_jupyter() and get_profile() is None:
-    # if no profile is loaded, load the default profile and raise a deprecation warning
-    from aiida import load_profile
+if is_running_in_jupyter():
+    from aiida.manage import get_profile
     from IPython.display import HTML, display
 
-    load_profile()
+    # load the default profile if no profile is loaded, and raise a deprecation warning
+    # this is a temporary solution to avoid breaking existing notebooks
+    # this will be removed in the next major release
+    if get_profile() is None:
+        # if no profile is loaded, load the default profile and raise a deprecation warning
+        from aiida import load_profile
+
+        load_profile()
+
+        profile = get_profile()
+        assert profile is not None, "Failed to load the default profile"
+
+        # raise a deprecation warning
+        warning = HTML(_WARNING_TEMPLATE.format(profile=profile.name, version="v3.0.0"))
+        display(warning)
+
+    from .static import styles
+    from .utils.loaders import load_css_stylesheet
 
-    profile = get_profile()
-    assert profile is not None, "Failed to load the default profile"
+    load_css_stylesheet(package=styles)
```

Comment:
To improve readability slightly, I'd move this to the top before the loading of the AiiDA profile.

Reply:
I think it had to be after. Otherwise, I got a warning message about not importing from the widgets base prior to loading the profile

Reply:
ah, of course, makes sense.